_generated/AzureTestPlanV0/task.json

```diff
@@ -125,6 +125,18 @@
       "defaultValue": false,
       "required": false,
       "helpMarkDown": "Fail the task if no result files are found."
+    },
+    {
+      "name": "mavenPOMFile",
+      "aliases": [
+        "mavenPomFile"
+      ],
+      "type": "filePath",
+      "label": "Maven POM file",
+      "defaultValue": "pom.xml",
+      "required": true,
+      "helpMarkDown": "Relative path from the repository root to the Maven POM file.",
+      "visibleRule": "testLanguageInput = Java-Maven"
```

Comment:
since testLanguageInput  is a multi select list, pls check if it works if only java maven is selected or multiple languages including maven is selected

Reply:
Removed, since it was not working as expected for multi-select input